src/main/scala/scala/tools/refactoring/implementations/oimports/ImportsOrganizer.scala

```diff
@@ -0,0 +1,79 @@
+package scala.tools.refactoring
+package implementations.oimports
+
+import scala.tools.nsc.Global
+
+abstract class ImportsOrganizers[G <: Global, U <: TreeToolbox[G]](val treeToolbox: U) {
```

Comment:
I guess I'm missing something: What is the outer class for? Why not just go with `ImportsOrganizer`?